proposals/lip-0046.md

```diff
@@ -161,7 +162,7 @@ This function returns `true` if `seedReveal` is valid input for the random seed
 ##### Execution
 
 ```python
-isSeedValidInput(generatorAddress, seedReveal):
+isSeedValidInput(generatorAddress: bytes, seedReveal: bytes) -> bool:
```

Comment:
I would use the type Address and add it to the type table (see the auth module LIP update)

```suggestion
isSeedValidInput(generatorAddress: Address, seedReveal: bytes) -> bool:
```

Reply:
Also please fix `false` --> `False` and `true` --> `True` in the whole LIP

Reply:
Done